src/platform-includes/sourcemaps/troubleshooting/javascript.mdx

```diff
@@ -1,167 +1,41 @@
-```
-
-This command uploads all JavaScript files in the current directory. The Artifacts page in Sentry should now look like:
-
-```
-~/scripts/script.js
-~/scripts/script.min.js
-~/scripts/script.min.js.map
-```
-
-Alternately you can specify which files to upload. For example:
+You can find the artifacts uploaded to Sentry by navigating to **[Settings] » Projects » Select your project » Source Maps**.
 
-```
-sentry-cli releases files VERSION upload-sourcemaps script.min.js script.min.js.map --url-prefix '~/scripts'
-```
-
-You can also upload it with the fully qualified URL. For example:
-
-```
-sentry-cli releases files VERSION upload-sourcemaps . --url-prefix 'http://localhost:8000/scripts'
-```
-
-### Using the API
-
-You can alternately [use our API](/api/releases/update-an-organization-release-file/) to upload artifacts, following the same naming convention explained here.
-
-```shell
-curl -X POST \
-  https://sentry.io/api/0/organizations/ORG_SLUG/releases/VERSION/files/ \
-  -H 'Authorization: Bearer AUTH_TOKEN' \
-  -H 'content-type: multipart/form-data' \
-  -F file=@script.min.js.map \
-  -F 'name=~/scripts/script.min.js.map'
-```
-
-### Using the `~`
-
-The `~` is used in Sentry to replace the scheme and domain. It is not a glob!
-
-`http://example.com/dist/js/script.js` will match `~/dist/js/script.js` or `http://example.com/dist/js/script.js`
-
-but will NOT match `~/script.js`.
-
-## Multiple Origins
+Make sure all of the necessary files are available. For Sentry to de-minify your stack traces you must provide both the minified files (for example, app.min.js) and the corresponding source maps. In case the source map files do not contain your original source code (`sourcesContent`), you must additionally provide the original source files. Alternatively, sentry-cli will automatically embed the sources (if missing) into your source maps.
 
-It's not uncommon for a web application to be accessible at multiple origins. For example:
+## Verify you are building sourcemaps
 
-- Website is operable over both `https` and `http`
-- Geolocated web addresses: such as `https://us.example.com`, `https://eu.example.com`
-- Multiple static CDNs: such as `https://static1.example.com`, `https://static2.example.com`
-- Customer-specific domains/subdomains
+Bundlers and tools (like `tsc`) that generate code generally require you to manually set options in order for source maps to be generated.
 
-In this situation, **identical** JavaScript and source map files may be located at two or more distinct origins. In this situation we recommend using our special tilde (`~`) prefix on paths.
+If you followed our tool-specific guides, verify that you configured your tool to emit source maps and that the source maps contain your original source code inside the `sourcesContent` field.
 
-So for example, if you have the following:
+## Verify your source files contain debug ID injection snippets
```

Comment:
I think the wording makes sense (+ see my comment regarding the code snippet). I'm just wondering for everyone who uses source maps w/o debug ids, this section will be confusing (they'll most likely read about this for the first time and identify that this code snippet isn't in their bundle, causing them to question their own sanity etc... 😅 ).

Should we add a sentence that redirects them to "Uploading without Debug IDs"? 

Something along the lines of 
```suggestion
## Verify your source files contain debug ID injection snippets

If you're uploading source maps by associating them to Sentry commits (e.g. Webpack Plugin v1), see [Uploading Source Maps without Debug IDs](LINK)

```

Reply:
I purposefully left out any references to the old process in my changes, simply because those who use the old process, likely already have working source maps and don't need the troubleshooting guide. I am honestly not sure if that is actually a good decision. In my mind it was worth a shot, just to reduce confusion/docs-spaghetty overall. Does that make sense?